docs/superpowers/specs/2026-03-12-yes-flag-design.md

```
@@ -0,0 +1,126 @@
+
+## CLI Interface
+
+**Flags:** `--yes` and `-y` (both equivalent, boolean, no value required)
+
+**Behavior:**
+- When present, prints warning to stderr at startup
+- Bypasses interactive Y/n prompts for bash commands
+- Security blocks (eval, exec, sudo, etc.) remain enforced
+
+**Warning message:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)